_scss/_navigation.scss

```diff
@@ -58,7 +62,9 @@
 
 .nav-secondary {
     background: $bg-secondary;
+    box-shadow: 0 1px 13px rgba(0, 0, 0, 0.24);
     height: 55px;
+    overflow: hidden;
```

Comment:
Adding `overflow: hidden;` here and in `.nav-container` makes the version select menu unusable (because almost all of it is hidden). 
![hidden-version-dropdown](https://cloud.githubusercontent.com/assets/4369687/25287889/023e4156-2678-11e7-91bd-824cafb75522.png)
@jsouth I've only ever tried to use that menu after this commit, so I'm not familiar with the "version select dropdown overflow issue" you mentioned in the PR, but what I'm seeing seems like a new issue caused by this.

If this isn't the right place to bring this to your attention, please direct me to the appropriate place.